templates/index.html

```diff
@@ -2,11 +2,67 @@
 <html>
 
 <head>
-    <title>PieceMeal</title>
-    <link rel="stylesheet" href="/static/index.css" />
+  <title>PieceMeal</title>
+  <link rel="stylesheet" type="text/css" href='../static/style.css'>
```

Comment:
No need to use relative paths here
```suggestion
  <link rel="stylesheet" type="text/css" href='/static/style.css'>
```